src/game/RemainingBox.js

```diff
@@ -12,10 +12,12 @@ const RemainingBox = props => {
         clickable,
         token,
         badPiece,
+        soloGame,
     } = props;
     const handleClick = async () => {
         if (idGame && boxValue > 0 && enabled && !selected && clickable) {
             await selectPiece(idGame, boxValue, token);
+            if (soloGame) await askAIMove(idGame);
```

Comment:
I think the front-end should not have to deal with other things than selecting a piece. But the changes will be too complicated for now.